packages/jaeger-ui/src/components/App/index.css

```diff
@@ -18,3 +18,84 @@ a {
 a:hover {
   color: var(--text-link-hover);
 }
+
+/* ============================================
+   Global Ant Design Overrides for Dark Mode
+   ============================================ */
+
+/* Input fields */
+.ant-input {
+  background-color: var(--surface-tertiary);
+  border-color: var(--border-default);
+  color: var(--text-primary);
```

Comment:
I think this is the wrong direction again. If anything we should be trying to __align__ with AntD theme support, not __work against it__. A change like this ^ undermines AntD's styling.

The ideal situation would be that we do not override so much in `ThemeConfig` in `ThemeProvider.tsx`, but instead:
  1. define minimum set in the theme provider where by setting up the Javascript properties we effectively override standard `var(--ant-...)` variables that are used by all standard AntD components
  2. we define our own variables by referencing the corresponding `--ant-xxx` variables wherever possible
  3. we use our own variables in our own components